angular-chart.js

```diff
@@ -202,12 +202,21 @@
     function getEventHandler (scope, action, triggerOnlyOnChange) {
       var lastState = null;
       return function (evt) {
-        var atEvent = scope.chart.getElementsAtEvent || scope.chart.getPointsAtEvent;
-        if (atEvent) {
-          var activePoints = atEvent.call(scope.chart, evt);
+        var atEvent = scope.chart.getElementAtEvent || scope.chart.getPointAtEvent;
+        var atEvents = scope.chart.getElementsAtEvent || scope.chart.getPointsAtEvent;
+        if (atEvents) {
+          var activePoints = atEvents.call(scope.chart, evt);
+          var activePointLst = atEvent.call(scope.chart, evt);
+          var activePoint;
+          if (atEvent && activePointLst.length) {
+            activePoint = activePointLst[0];
+          } else {
+            activePoint = undefined;
+          }
           if (triggerOnlyOnChange === false || angular.equals(lastState, activePoints) === false) {
             lastState = activePoints;
-            scope[action](activePoints, evt);
+            // TODO clean param order or find another idea
```

Comment:
I don't like the situation, third argument is the selected point, but first is list of all point.
But if we change order, many people will be angry because API change.